AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/deploy/boundary/DeploymentBoundary.java

```diff
@@ -264,6 +274,18 @@ public boolean isPossible() {
         return new Tuple<>(deployments, totalItemsForCurrentFilter);
     }
 
+    private boolean isH2() {
```

Comment:
Duplicate Code with: https://github.com/liimaorg/liima/blob/0daea903e6b6a0962b21870957f3f44aabafe9bf/AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/domain/applist/ApplistScreenDomainServiceQueries.java#L90
Should be moved to some commen place.